lib/manager/util/version.js

```diff
@@ -170,3 +171,15 @@ export function getVersionValidationSummaryByFilterStrategy (
 export const versionsLastUpdatedComparator = (a: FeedVersion, b: FeedVersion) => {
   return a.updated - b.updated
 }
+
+/**
+ * Formats a difference with a leading + sign, or as '=' if the difference is zero,
+ * and converts large numbers into nearest thousands unit (e.g. 25000 => 25k).
+ */
+export function formatDelta (
+  delta: number
+): string {
+  return delta === 0
+    ? '='
```

Comment:
Not crazy about the `=` for "no change", but I don't have a great alternative at the moment.

Reply:
One alternative would be to display nothing.